api.go

```diff
@@ -1 +1,20 @@
 package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+func UserConfigRecieveHandler(c echo.Context) error {
+	var config SocketConfig
+	if err := c.Bind(&config); err != nil {
+		return c.JSON(http.StatusBadRequest, map[string]string{"error": "Invalid input"})
```

Comment:
Make a function to return errors 
return WriteErrorResponse(c echo.Context, err error, msg string) error {
      return c.JSON(code, map[string]string{"msg":msg,"error":err.Error()})
      
      Something like this